graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/NodeQueryExtensions.java

```diff
@@ -56,21 +58,29 @@ private Workspace(String workspace) {
         public String getValue() {
             return workspace;
         }
+
+        public static class DefaultWorkspaceSupplier implements Supplier<Object> {
+
+            @Override
+            public Workspace get() {
+                return EDIT;
+            }
+        }
```

Comment:
damn java you verbose language, 7 lines just to get a default value

Reply:
it's actually to provide a class (needed by GraphQL) of a _Supplier_, not just a default value ;)